ktor-client/ktor-client-apache/jvm/src/io/ktor/client/engine/apache/ApacheResponseConsumer.kt

```diff
@@ -45,8 +45,9 @@ internal class ApacheResponseConsumer(
     override fun consumeContent(decoder: ContentDecoder, ioctrl: IOControl) {
         check(!waiting.value)
 
-        var result = 0
+        var result: Int
```

Comment:
Could you at least add a comment about how this change fixes the problem or add some information in the commit message?

Reply:
I manage to add a test. Is this enough?